circuit-lib/circuit-lib.circom/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@lightprotocol/circuit-lib.circom",
+  "version": "0.1.0-alpha.1",
+  "description": "",
+  "license": "GPL-3.0",
+  "scripts": {
+    "build-masp": "sh scripts/buildCircuit.sh Masp",
+    "build-app": "sh scripts/buildCircuit.sh App",
+    "test-light-circuits": "ts-mocha --resolveJsonModule ./tsconfig.json -t 100000000 tests/**.ts --exit",
+    "test-build-light-circuits": "yarn build-all && ./scripts/checkBuildCircuit.sh",
+    "test": "yarn test-light-circuits && yarn test-build-light-circuits",
+    "format": "prettier --write \"tests/**/*.{ts,js,circom}\"",
+    "build-all": "yarn build-app 4 && yarn build-masp 2 && yarn build-masp 10",
+    "lint": "yarn prettier \"tests/**/*.{ts,js}\" --check",
+    "build": "yarn tsc"
+  },
+  "author": "",
+  "devDependencies": {
+    "@types/chai": "^4.3.0",
+    "@types/mocha": "^10.0.1",
+    "@types/node-fetch": "^2.6.2",
+    "chai": "^4.3.7",
+    "@lightprotocol/zk.js": "file:../../light-zk.js",
```

Comment:
good catch I will use the linked package
I had issues here because I wanted to import both zk.js and cli as dev dependencies.
- the issue was that cli is build with linked zk.js, somehow this caused an issue
- I switched the [cli dependency to a relative path](https://github.com/Lightprotocol/light-protocol/pull/251/files/728841d04bb2022e5238b197c194bd8391755751#diff-05568b9d7b02e4ddb0f0e441194d1299e9eb360ddb9f3ab583c6b56f3434ff02R2) to get around it and it should be enough

in essence:
- switched zk.js to linked version
- cli is relative path